### ISSUE TITLE
Step size for wind turbines is way too small: increase to 1 unit

### ISSUE THREAD

**Issue by AlexanderWirtz** [quintel/etmodel#1885]
Currently, both 800 MWe power plants and 3 MWe wind turbines both have sliders associated with them that use a step size of `0.1`. This seems way too small for wind turbines.

To me it makes sense to increase the wind turbine step size to 1 (i.e. 3 MWe). This will have some fallout for existing scenario's but numerically, it shouldn't make much of a difference, I expect. 

Of course, the same may be true for some other technologies. This is an issue that mostly affects the optimizer, as this doesn't recognize units. 

@ChaelKruip assigning you, but feel free to assign someone else, like an intern
@jorisberkhout feel free to offer your opinion on what a good step size might be.

**Comment by ChaelKruip** [comment]
@AlexanderWirtz good point! Does this need to be fixed before RLI optimisation?

**Comment by AlexanderWirtz** [comment]
No, because we will probably not do another optimization. Let's have someone (an intern?) have a look again at sensible step sizes for the sliders.
